FlashDevelop/MainForm.cs

```diff
@@ -903,10 +903,21 @@ public void CheckForUpdates()
         public void InitializeWindow()
         {
             this.WindowState = this.appSettings.WindowState;
-            Point position = new Point(this.appSettings.WindowPosition.X, this.appSettings.WindowPosition.Y);
-            if (position.X < -4) position.X = 0;
-            if (position.Y < -25) position.Y = 0;
-            this.Location = position;
+            Rectangle bounds = new Rectangle(this.appSettings.WindowPosition, this.appSettings.WindowSize);
+            Boolean validPosition = false;
```

Comment:
Maybe adding `bounds.Inflate(-4, -25)` here could be better, since previously it was like that.